Improved external reference documentation. (#6)
GENIVI/ramses-composer-docs: external_references/manual.md

Likewise, you can delete resources manually. But Ramses Composer can do this for you and detect and delete all resources currently not used by any other objects can be automatically. Select *Delete Unused Resources* from the context menu in the *Resources* view. This will apply to both External References and normal resource objects. It is a good idea to save your scene beforehand, just in case more things disappear than expected.

## Splitting existing Projects into Modules

> To split an existing scene into multiple modules and integrate them into the original master scene using external references the following approximate workflow can be used

Reviewer: ```suggestion
To split an existing scene into multiple modules and integrate them into the original master scene using external references the following approximate workflow can be used:
```

1. Prefab conversion: create Prefabs for the parts of the scenegraph that should be placed into a module scene.

> 2. Creating module scene: Deep cut the Prefabs and paste them into a newly created scene for the module. This may be more convenient when using multiple RamsesComposer instances. The pasting into the module scene needs to use normal paste and not paste as external reference.

Reviewer: ```suggestion
2. Creating module scene: Deep cut the Prefabs and paste them into a newly created scene for the module. An easy way to do this is to use multiple RamsesComposer instances. When pasting into the module scene, use normal paste and not `paste as external reference`.
```
